app/controllers/webhooks_controller.rb

```diff
@@ -8,6 +8,8 @@ class WebhooksController < ApplicationController
 
   def index
     @webhooks = current_project.webhooks
+    @outbound_webhooks = current_project.outbound_webhooks
+    @new_outbound_webhook = OutboundWebhook.new
```

Comment:
The normal webhook doesn't need it because there are no errors rendered, and we don't care about losing the form data.  It's not necessary if we don't want to preserve form data on error.  I can move it in the view, sure.